public/opensearch.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+  <ShortName>DUB</ShortName>
+  <Description>Search for dub packages</Description>
+  <Image type="image/png">https://code.dlang.org/images/logo-small.png</Image>
+  <Url type="text/html" method="get" template="https://code.dlang.org/search?q={search-query}"/>
```

Comment:
I'm not familiar with the OpenSource format so I may be wrong on this, but [it looks](https://github.com/dewitt/opensearch/blob/master/opensearch-1-1-draft-6.md#the-searchterms-parameter) like this should be "{searchTerms}":

```suggestion
  <Url type="text/html" method="get" template="https://code.dlang.org/search?q={searchTerms}"/>
```

Reply:
Thanks, I missed.